CleanWeather/Support/en.lproj/Localizable.strings

```diff
@@ -6,25 +6,27 @@
   Copyright © 2019 vandermesis. All rights reserved.
 */
 
-// Errors
-"error" = "Error";
-"unknownError" = "Unknown Error";
-"decodingError" = "Decoding Error";
-"encodingError" = "Encoding Error";
-"serializerError" = "Serializer Error";
-"userDefaultsError" = "User Defaults Error";
-"dataSourceError" = "Cell DataSource Error";
-"apiError" = "Api responded with Error";
-"invalidURL" = "Invalid URL";
-"missingApiResponse" = "Api not responded";
-"missingApiData" = "No data for that request";
+// CityList
+"CityListControllerTitle" = "CleanWeather";
```

Comment:
I still meant to have dots between
CityList.ControllerTitle

And maybe get rid of 'Controller' as it would be everywhre:
CityList.Title